RevenueCatUI/Templates/Components/Button/ButtonComponentViewModel.swift

```diff
@@ -39,6 +39,11 @@ public class ButtonComponentViewModel {
         )
     }
 
+    func onClick() {
+        // swiftlint:disable:next todo
+        // TODO Handle the configured Action.
+    }
```

Comment:
I mentioned this in a PR further down the chain but I would probably name this to `performAction()`since it could be a click or a tap 😇 Just a nit of mine 😅

Reply:
Oh yes, great point. I'll rename it in the other PR, since this function is moved in that PR as well.